### PR TITLE
MNT Run module-standardiser

### PR DESCRIPTION
**Pull request by emteknetnz**
This pull-request was created automatically by [module-standardiser](https://github.com/silverstripe/module-standardiser)

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
This PR adds the phpstan stuff, but no phplinting job is in the CI run. We should either not include phpstan stuff for recipes, or make sure to include PHP linting in their CI runs.

I'd suggest not including it - everything in a recipe is included in projects and I don't think we want projects to have our idiosyncratic module-related standards enforced.

**Comment by emteknetnz** [comment]
Have removed and updated module-standardiser to not include it in future 
- https://github.com/silverstripe/module-standardiser/pull/50

File: composer.json [hunk 1]
```diff
@@ -47,7 +47,9 @@
         "silverstripe/frameworktest": "^1",
         "silverstripe/graphql-devtools": "1.x-dev",
         "silverstripe/recipe-testing": "^3",
-        "mikey179/vfsstream": "^1.6"
+        "mikey179/vfsstream": "^1.6",
+        "silverstripe/standards": "^1",
+        "phpstan/extension-installer": "^1.3"
```
Comment (ChloeSartorelli) [inline composer.json]:
```suggestion
        "mikey179/vfsstream": "^1.6"
```

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**